sortinghat/cli/cmds/lock.py

```diff
@@ -43,22 +43,22 @@ def lock(ctx, **extra):
 @click.argument('uuid')
 @click.pass_obj
 def add(client, uuid):
-    """Add a lock to a unique identity so it cannot be modified.
+    """Add a lock to a individual so it cannot be modified.
 
-    This command adds a lock to the unique identity identified
+    This command adds a lock to the individual identified
     by <uuid>. Therefore, this and its related entities such
     as identities, enrollments or the profile cannot be
     modified.
 
-    UUID: identifier of the unique identity which will be locked
+    UUID: identifier of the individual which will be locked
     """
     with connect(client) as conn:
         _lock_identity(conn, uuid=uuid)
         display('lock.tmpl', uuid=uuid, unlocked=False)
 
 
 def _lock_identity(conn, **kwargs):
-    """Run a server operation to lock a unique identity."""
+    """Run a server operation to lock a individual."""
```

Comment:
Same as above (`an individual`). Moreover, I think this should be renamed too to `_lock_individual`.